src/components/OverlayImage.tsx

```diff
@@ -39,6 +38,30 @@ const OverlayImage = ({ url, opacity, scaling, isLocked, showDifference }: Overl
     return () => window.removeEventListener("mousemove", handleMouseMove);
   }, [mouseDown]);
 
+  // Ability to move zeplin overlay with Shift + Arrow keys
```

Comment:
I think we can remove this comment. Also, I'm not used to seeing our company name lowercase 😬